Content/ua_20200713_Feats-2020.js

```diff
@@ -3,9 +3,370 @@ RequiredSheetVersion(13);
+	url: "https://media.wizards.com/2020/dnd/downloads/UA2020_Feats.pdf",
+	date: "2020/07/13"
+};
+
+FeatsList["artificer initiate"] = {
+	name: "Artificer Initiate",
+	source: [
+		["UA:F20", 1]
+	],
+	description: "I learn two cantrips and one 1st-level spell of my choice from the artificer's spell list. I can cast the 1st-level spell at its lowest level once per long rest without using a spell slot. Int is my spellcasting ability. I gain proficiency in one artisan's tool that I can use as focus.",
+	spellcastingBonus: [{
+		name: "Artificer cantrip",
+		spellcastingAbility: 4,
+		class: 'artificer',
+		level: [0, 0],
+		atwill: true,
+		times: 2
+	}, {
+		name: "Artificer 1st-level spell",
+		class: 'artificer',
+		level: [1, 1],
+		oncelr: true
+	}],
+	toolProfs: [
+		["Artisan's tools", 1]
+	]
+};
+FeatsList["chef"] = {
+	name: "Chef",
+	source: [
+		["UA:F20", 1]
+	],
+	description: "During a short rest, I can make food for 4 + my proficiency bonus creatures; if they eat it and spend 1+ Hit Die they regain 1d8 HP. During a long rest I can make snacks equal to my proficiency bonus that give my proficiency bonus temp HP [+1 Constitution or Wisdom]",
+	toolProfs: ["Chef's utensils"],
+	improvements: "Chef (feat): +1 Constitution or Wisdom;"
+};
+FeatsList["crusher"] = {
+	name: "Crusher",
+	source: [
+		["UA:F20", 1]
+	],
+	description: "Once per turn when I deal bludgeoning damage to a target, I can move it 5 feet if it is less than 1 size larger than me. If I deal bludgeoning damage on a critical hit to a target attack rolls against it are with advantage until end of my next turn. [+1 Strength or Dexterity]",
+	improvements: "Crusher (feat): +1 Strength or Dexterity;"
+};
+// NOT DONE //
+FeatsList["eldritch adept"] = {
+	name: "Eldritch Adept",
+	source: [
+		["UA:F20", 1]
+	],
+	description: "I learn one Eldritch Invocation option of my choice from the Warlock class. If it has a prerequisite, I can only take it if I am a warlock and meet the prerequisite. I can replace the invocation whenever I gain a level.",
+	prerequisite: "Spellcasting or Pact Magic feature",
+	prereqeval: function (v) {
+		return What('Class Features').toLowerCase().indexOf('spellcasting') === -1 || What('Class Features').toLowerCase().indexOf('pact magic') === -1;
+	}
+};
+FeatsList["fey touched"] = {
+	name: "Fey Touched",
+	source: [
+		["UA:F20", 2]
+	],
+	description: "I learn Misty Step and one 1st level spell of my choice from the divination or enchantment school of magic. I can cast each once per lost rest without expending a spell slot, and can cast them if I have the slot to do so.",
+	choices: ["Intelligence", "Wisdom", "Charisma"],
+	"intelligence": {
+		description: "I learn Misty Step and one 1st level spell of my choice from the divination or enchantment school of magic. I can cast each once per lost rest without expending a spell slot, and can cast them if I have the slot to do so. Intelligence is my spellcasting ability [+1 Intelligence]",
```

Comment:
Perhaps "Charisma is my spellcasting ability for these spells" to make it clear it doesn't wholly affect the class.

Reply:
This actually was an issue fitting it on the Colorful sheet, as there is less room because of the non-monospaced font. I have foudn a way to update the spell description to be less wordy and fit that text.